website/content/specification/objects/scenario.md

```diff
@@ -7,20 +7,85 @@ title: "Scenario"
 A scenario describes the conditions surrounding the possible use of a vulnerability. A Vulnerability must have a least one Scenario, with multiple possible Scenarios being common. A single Vulnerability can likely be exploited by many different approaches with possible varying impacts. Multiple Scenarios can be used to describe these variations. For example, a remote exploit could rely on user interaction to be downloaded whilst a local attacker could use the same vulnerability to obtain the same or similar impact.
 
 ## Properties
-- **id** (one): A globally unique identifier for the scenario that distinguishes it from other scenarios related to the same vulnerability.
-- **hasName** (zero or one): A name or label to assist in identifying a given scenario in the context of the containing Vulnerability. This name should be unique across all sibling scenarios.
-- **requiresAttackTheater** (one): Attack Theater is the area or place from which an attack must occur. Each separate theater represents varying levels of implied trust and attack surface. (See [Theater](../../values/theater))
-- **evidencedBySource** (one or many):  [Resource Reference](../../values/resource-reference) will assist in proving a Vulnerability Scenario is legitimate. 
-- **hasExploitedWeakness** (one): The weakness causing the Vulnerability. When choosing a value, the most applicable weakness should be selected. (See [Exploited Weakness](../../values/exploited-weakness))
 
+An *scenario* has the following properties.
+
+### Identifier
+
+{{%usa-tag%}}Name{{%/usa-tag%}} `id`
+{{%usa-tag%}}Cardinality{{%/usa-tag%}} one
+{{%usa-tag%}}Description{{%/usa-tag%}} A globally unique identifier for the *scenario*.
+
+The action identifier distinguishes the *scenario* from other *scenarios* related to the same *vulnerability*.
+
+This identifier MUST be a version 4 (random) or 5 (SHA-1 based) Universally Unique Identifier (UUID) as defined by [RFC 4122](https://www.rfc-editor.org/rfc/inline-errata/rfc4122.html).
+
+### Name
+
+{{%usa-tag%}}Name{{%/usa-tag%}} `hasName`
+{{%usa-tag%}}Cardinality{{%/usa-tag%}} zero or one
+{{%usa-tag%}}Description{{%/usa-tag%}} A name or label to assist in identifying a given *scenario* in the context of the containing *vulnerability*.
+
+A given *scenario* name MUST be unique across all sibling *scenarios*.
+
+The `value` MUST be based on the lexical space of a string as defined by [ECMA-404 2nd edition, section 9](https://www.ecma-international.org/wp-content/uploads/ECMA-404_2nd_edition_december_2017.pdf).
+
+### Required Attack Theater
```

Comment:
Prefer just "Attack Theater", although I won't die on that hill

Reply:
```suggestion
### Attack Theater
```